autogpt/config/config.py

```diff
@@ -136,6 +136,16 @@ def __init__(self) -> None:
             self.plugins_allowlist = []
         self.plugins_denylist = []
 
+        # event dispatcher configuration
+        self.event_dispatcher_enabled = os.getenv("EVENT_DISPATCHER_ENABLED", "False")
+        self.event_dispatcher_protocol = os.getenv("EVENT_DISPATCHER_PROTOCOL")
+        self.event_dispatcher_host = os.getenv(
+            "EVENT_DISPATCHER_HOST",
+        )
+        self.event_dispatcher_endpoint = os.getenv("EVENT_DISPATCHER_ENDPOINT")
+        self.event_dispatcher_port = os.getenv("EVENT_DISPATCHER_PORT")
+        self.event_dispatcher_mute = os.getenv("EVENT_DISPATCHER_MUTE", "True")
+
```

Comment:
Are defaults needed for the rest of these

Reply:
set